src/code/z_player_lib.c

```diff
@@ -892,10 +914,16 @@ Gfx* gPlayerLeftHandOneHandSwordDLs[2 * PLAYER_FORM_MAX] = {
     gLinkHumanLeftHandHoldingKokiriSwordDL,
 };
 
-Gfx* D_801C018C[][2] = {
-    { gLinkHumanLeftHandHoldingKokiriSwordDL, gLinkHumanLeftHandHoldingKokiriSwordDL },
-    { gLinkHumanLeftHandHoldingRazorSwordDL, gLinkHumanLeftHandHoldingRazorSwordDL },
-    { gLinkHumanLeftHandHoldingGildedSwordDL, gLinkHumanLeftHandHoldingGildedSwordDL },
+Gfx* D_801C018C[] = {
+    // EQUIP_VALUE_SWORD_KOKIRI
+    gLinkHumanLeftHandHoldingKokiriSwordDL,
+    gLinkHumanLeftHandHoldingKokiriSwordDL,
+    // EQUIP_VALUE_SWORD_RAZOR
+    gLinkHumanLeftHandHoldingRazorSwordDL,
+    gLinkHumanLeftHandHoldingRazorSwordDL,
+    // EQUIP_VALUE_SWORD_GILDED
+    gLinkHumanLeftHandHoldingGildedSwordDL,
+    gLinkHumanLeftHandHoldingGildedSwordDL,
```

Comment:
I think there's scope for confusion here now that it's two of each, is this a lod thing?

Reply:
Of course there's scope for confusing here, even I'm confused what's going on here.
If I had to guess it is a lod-related thing, and most of the player lod stuff are leftovers from OoT, since MM uses the same model for both